### PR TITLE
Updates metadata template

### PR DESCRIPTION
**Pull request by liz-fw**
Splits metadata example into two files: project metadata and datatable metadata. Also updates this in the Banet example folder. Updates documentation in vignette to reference the separate tables.

### CODE REVIEW

File: vignettes/template-doc.Rmd [hunk 1]
```diff
@@ -19,15 +19,15 @@ will find the following assets for recording your projects metadata:
 
 * methods.docx
 * abstract.docx
-* metadata.xlsx
+* project-metadata.xlsx
```
Comment (ErinCain) [inline vignettes/template-doc.Rmd:22]: these changes look good! Please rebuild pkgdown site so that the article is updated on the site.

Comment (ErinCain) [inline vignettes/template-doc.Rmd:22]: Update metadata template materials in the aws bucket (linked above as metadata_template materials) 
You can just update the existing docs in there with the up-to-date metadata template example and text. There is a pdf metadata template documentation in there that is generated by running this vignette but saving as a PDF (update output section of md frontmatter). Let me know if you have questions on this.

**Review by ErinCain — COMMENTED** [review]
Looks good Liz. Just a couple of remaining pieces here. Please update metadata_template materials on the s3 site and rebuild pkgdown site! Thanks (:

File: vignettes/template-doc.Rmd (hunk 1 above)
Comment (liz-fw) [reply vignettes/template-doc.Rmd:22]: i added the updated metadata template.zip (including the updated pdf) to the aws bucket so it should be linking correctly from the site!

**Review by ErinCain — APPROVED** [review]

**Pull request merged**